Dockerfile

```diff
@@ -13,14 +13,16 @@ ADD requirements.txt /opt/atomicapp/
 
 WORKDIR /opt/atomicapp
 
-RUN pip install .
+RUN python setup.py install
+RUN pip install -r ./requirements.txt
```

Comment:
Does this look good ...RUN yum install -y gcc && python setup.py install && pip install -r ./requirements.txt && yum clean all ?